ui/tests/unit/adapters/node-test.js

```diff
@@ -20,7 +20,6 @@ module('Unit | Adapter | Node', function(hooks) {
     this.server.create('allocation', { id: 'node-1-2', nodeId: 'node-1' });
     this.server.create('allocation', { id: 'node-2-1', nodeId: 'node-2' });
     this.server.create('allocation', { id: 'node-2-2', nodeId: 'node-2' });
-    this.server.logging = true;
```

Comment:
I was wondering about this!

Reply:
This deletion was special enough to get its own commit 🤓 

It was already in the code (as evident by the diff) and I just happened to notice it spewing log lines in the test runner. 

If you aren't familiar with `this.server.logging = true`, it logs every HTTP request that goes through Pretender. Great for debugging, super noisy though.

Reply:
yeah, what do you think of [this kind of solution](https://github.com/backspace/prison-rideshare-ui/blob/primary/mirage/config.js#L7) that lets you add a query parameter to turn on logging? I like it more than having to edit code 😆

Reply:
ooOOOoo, that's quite neat indeed. I like that. And nothing about the qunit runner removes that query param in an annoying manner?

Reply:
I haven’t had any trouble with it so far! 🎉

Reply:
+1 to doing this. We do this in consul lots, but we use `document.cookies` instead. Using cookies means settings stick around when the URL changes, which is handy. Then we use Web Inspector to edit the cookies, which turns the Web Inspector into kind of a semi 'dev preferences' panel.

Actually, we didn't ever add a setting to turn HTTP logging on and off (we use it more to control mocks/doubles), but I think I'd add that also as we have some noisy logging going on aswell which I don't mind but other folks might.